### PR TITLE
Fixed or removed broken translations (bsc#1088113)

### PR DESCRIPTION
**Pull request by lslezak**
This is related to https://bugzilla.suse.com/show_bug.cgi?id=1088113

Ideally we should somehow solve the problem with the `format` placeholders. However it's not trivial so for now I have simply checked all texts by a script and fixed or removed the broken translations.

- I have fixed obvious typos or wrong formats
- I have removed obviously invalid translations
- When I was not sure if the translation is correct then I removed it too. IMHO it's better to have a correct English text than invalid/broken translation. Moreover the translators can simply translate again.
- The parser I used (the Ruby [PoParser](https://github.com/arashm/PoParse)) does not serialize 1:1 when saving back to the original file, some white space or some empty tags might be lost. But I think it's acceptable, it only makes this diff a bit longer for some files.

I have attached the checking script, do not review it tool strictly, it's a quick hack to solve the translation issues ASAP. :wink:

### CODE REVIEW

File: po/fonts/ca.po [hunk 1]
```diff
@@ -517,7 +515,7 @@ msgstr ""
 
 #: src/lib/fonts/fonts-dialog.rb:1048
 msgid "<li><b>%{name}: </b>%{help}</li>"
-msgstr "<li><b>%{nom}: </b>%{ajuda}</li>"
+msgstr "<li><b>%{name}: </b>%{help}</li>"
```
Comment (mvidner) [inline po/fonts/ca.po:518]: This message is a trap :)

File: po/installation/ar.po [hunk 2]
```diff
@@ -160,7 +160,7 @@ msgid ""
 "differ from those in the manual.\n"
 msgstr ""
 "لا يستوفي الكمبيوتر الخاص بك كل المتطلبات \n"
-"للتثبيت الرسومي. هناك إما أقل من 1% في ذاكرة MB\n"
+"للتثبيت الرسومي. هناك إما أقل من 1% في ذاكرة %1 MB\n"
```
Comment (mvidner) [inline po/installation/ar.po]: Good catch, but your correction is still wrong because it includes both %1 and 1%. I'll fix it.

**Comment by stanislav-brabec** [comment]
I am merging it now, as po files pull requests tend to break quickly. Many thanks for your work.

File: po/services-manager/id.po [hunk 3]
```diff
@@ -173,8 +171,7 @@ msgstr ""
 msgid ""
 "Service %service will be %toggled and port in firewall will be %switched "
 "%link"
-msgstr ""
-"Layanan %service akan %toggled dan port pada firewall akan %diganti %tautan"
+msgstr "Layanan %service akan %toggled dan port pada firewall akan %switched %link"
```
Comment (mvidner) [inline po/services-manager/id.po:174]: @lslezak This is nonsense and can never work, %s is a string but %t raises "ArgumentError: malformed format string - %t"

Comment (mvidner) [reply po/services-manager/id.po:174]: It looks like the proposal code is never called anyway.

**Pull request merged**